gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/launcher/ExecuteGobblinJobLauncher.java

```diff
@@ -83,10 +83,12 @@ public ExecuteGobblinJobLauncher(
   public void submitJob(List<WorkUnit> workunits) {
     try {
       Properties finalProps = adjustJobProperties(this.jobProps);
+      // Initialize workflowId to be used by cancel workflow.
+      this.workflowId = Help.qualifyNamePerExecWithFlowExecId(WORKFLOW_ID_BASE, ConfigFactory.parseProperties(jobProps));
```

Comment:
this should be `finalProps`, right?

Reply:
nice catch, finalProps was added recently and I missed it in master merge